docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+  - biconomyPaymasterApiKey(`string`) OR paymaster (`IPaymaster`): one can either pass paymaster API key or the paymaster instance.
+  - entryPointAddress (`string`): default entry point address will be used if not passed, otherwise the passed address will be used.
+  - defaultValidationModule (`BaseValidationModule`): The default validation module to start with ( ECDSA ).
+  - activeValidationModule (`BaseValidationModule`): The active validation module to start with ( either ECDSA or Multi-chain ).
+  - rpcUrl (`string`): RPC URL of the chain
+  - index (`number`): index to create multiple smart accounts for an EOA
+
+**Returns**
+
+- `smartAccount` (`object`): An instance of the Biconomy Smart Account V2.
+
+:::info
+Building on Chiliz Mainnet or the Spicy Testnet? Note that the entry point address on this is different as it was deployed by us on the Biconomy team. The address of the entry point is : [0x00000061FEfce24A79343c27127435286BB7A4E1](https://scan.chiliz.com/address/0x00000061FEfce24A79343c27127435286BB7A4E1/contracts#address-tabs)
+:::
+
+## Smart Account Get Methods
+
+### getAccountAddress( )
+
+This method is used to retrieve the account address associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+const address = await smartAccount.getAccountAddress();
+```
+
+**Returns**
+
+- address (`Promise<string>`): A Promise resolving to the account address associated with the smartAccount instance.
+
+### getNonce( )
+
+This method is used to retrieve the nonce associated with the `smartAccount` instance.
+
+**Usage**
+
+```jsx
+
+const nonce = await smartAccount.getNonce();
+console.log(nonce.toNumber());
+```
+**Parameters**
+- nonceKey(`number`): one can also pass the optional parameter nonceKey in the case of two-dimensional nonces
+
+**Returns**
+
+- address (`Promise<BigNumber>`): A Promise resolving to BigNumber indicating the nonce of the smart account.
+
+### index
+
+This is used to retrieve the index of the current active smart account.
+
+**Usage**
+```jsx
+const index = smartAccount.index;
+```
+
+**Returns**
+- index (`number`): A number indicating the index of current active smart account.
+
+## UserOp Methods
+
+### buildUserOp( )
+This method is used for configuring and setting up all properties of the `userOp` object.  It involves preparing and populating the userOp with necessary information and parameters required for execution by bundler.
+
+**Usage**
+
+For example, in the context of creating a userOp for an `addComment` transaction, an instance of the contract is created, then a basic transaction object is created that holds the necessary address and data from the transaction. Finally, a userOp is created using the Smart Account's `buildUserOp` method⁠.
```

Comment:
```suggestion
For example, in the context of creating an user operation for an `addComment` transaction, an instance of the contract is created, then a basic transaction object is created that holds the necessary address and data for the transaction. Then, a partial userOp is built using the Smart Account's `buildUserOp` method⁠. Now this can be signed and sent to the bundler.
```